src/store/actions/getNFTAssets.ts

```diff
@@ -22,20 +22,50 @@ export const getNFTAssets = async (
 
   const nft = await client.nft.fetch();
 
-  const nftAssets: NFTAsset[] = [];
-  nft.assets.forEach((asset: any) => {
-    if (state.starredNFTs) {
-      asset['starred'] = state.starredNFTs.filter(
-        (item) => item.asset_contract.address === asset.asset_contract.address && item.id === asset.id
-      ).length
-        ? true
-        : false;
-      nftAssets.push(asset);
+  const oldAssets: NFTAsset[] = state.nftAssets;
```

Comment:
What is the purpose of this function?  Can you describe input and outputs?
It's really hard to understand and I feel there is a much clearer way to write what this is doing